diff --git a/targetcli/ui_root.py b/targetcli/ui_root.py
index 79ee985..4e40aa3 100644
--- a/targetcli/ui_root.py
+++ b/targetcli/ui_root.py
@@ -160,7 +160,7 @@ class UIRoot(UINode):
 
                 try:
                     with open(universal_prefs_file) as prefs:
-                        backups = [line for line in prefs.read().splitlines() if re.match('^max_backup_files\s*=', line)]
+                        backups = [line for line in prefs.read().splitlines() if re.match(r'^max_backup_files\s*=', line)]
                         if max_backup_files < int(backups[0].split('=')[1].strip()):
                             max_backup_files = int(backups[0].split('=')[1].strip())
                 except:
